src/main/kotlin/com/zepben/evolve/services/diagram/DiagramServiceInstanceCache.kt

```diff
@@ -10,7 +10,7 @@ package com.zepben.evolve.services.diagram
 import com.google.common.collect.Interners
 import com.zepben.evolve.cim.iec61970.base.diagramlayout.DiagramObjectPoint
 
-@Suppress("UnstableApiUsage")
+// TODO find out if this is used
 class DiagramServiceInstanceCache {
```

Comment:
There doesn't seem to be a usage for either `DiagramServiceInstanceCache` or `NetworkServiceInstanceCache` in Zepben's repos. Is this meant to be public API?

Reply:
Removed both classes.